tests/UnifiedSpecTests/EventCollector.php

```diff
@@ -119,18 +119,17 @@ private function handleCommandMonitoringEvent($event): void
             'name' => self::getEventName($event),
             'observedAt' => microtime(true),
             'commandName' => $event->getCommandName(),
-            'connectionId' => $event->getServerConnectionId(),
-            'requestId' => $event->getRequestId(),
+            'databaseName' => $event->getDatabaseName(),
             'operationId' => $event->getOperationId(),
+            'requestId' => $event->getRequestId(),
+            'server' => $event->getHost() . ':' . $event->getPort(),
+            'serverConnectionId' => $event->getServerConnectionId(),
+            'serviceId' => $event->getServiceId(),
         ];
 
         /* Note: CommandStartedEvent.command and CommandSucceededEvent.reply can
          * be omitted from logged events. */
 
-        if ($event instanceof CommandStartedEvent) {
-            $log['databaseName'] = $event->getDatabaseName();
-        }
```

Comment:
Why do you remove this?

Reply:
It's now added unconditionally a few lines up.

This original logic predated PHPC 1.19 adding the property to all APM events in [PHPC-2255](https://jira.mongodb.org/browse/PHPC-2255).